NOTICE.md

```diff
@@ -0,0 +1,48 @@
+# Notices for Eclipse Tractus-X
+
+This content is produced and maintained by the Eclipse Tractus-X project.
+
+* Project home: <https://projects.eclipse.org/projects/automotive.tractusx>
+
+See the AUTHORS file(s) distributed with this work for additional information regarding authorship.
```

Comment:
AUTHORS seems to be missing. IMHO this is not a must to add the AUTHORS.md file. TRGs does not request for it.